scss/_carousel.scss

```diff
@@ -104,7 +104,7 @@
   @include hover-focus {
     color: $carousel-control-color;
     text-decoration: none;
-    outline: 0;
+    outline: 2px dotted #000;
```

Comment:
Divergence from bootstrap need to be commented
- add // boosted mod at the end of line
- don't use color code in scss file use var instead currently you could use $carousel-control-color who's coherent